### DISCUSSION TITLE
Unable to Create Additional Section That Behaves Like Docs

### DISCUSSION THREAD

**Discussion by ghost** [imfing/hextra#18] [Q&A] [answered]
I have two documentation sections:

1. Concepts - located in `/content/docs/`
2. Exercises - located in `/content/exercises/`

When I navigate to Concepts everything is displayed as expected. 

But when I navigate to Exercises and click on a page: 

+ I don't see the navigation bar on the left
+ There are no breadcrumbs
+ Title is aligned in the center instead of on the left

[This](https://codeberg.org/mishra/ferris.rs) is the repository in question.

Steps to reproduce:
+ Start Hugo server with draft pages
+ Go to the homepage
+ Navigate to Exercises > Airport Robot

Please let me know how I can ensure both these sections behave like each other.

**Comment by imfing** [comment, marked as answer]
To make a section resemble the `docs` layout, add the `type: docs` field to the frontmatter of the corresponding pages. Specifically, for pages located in the `/content/exercises` directory, include the following frontmatter:
```yaml
---
title: Your Title Here
type: docs
---
```
This will ensure those pages adopt the 'Docs' layout.

**Reply by imfing** [reply]
It's not ideal since the theme needs to incorporate different sections like docs, blog and single pages.
I will figure out a better solution in the future to make the elements more configurable through the page's frontmatter.

**Reply by billux** [reply]
I'm also in this case, with multiple top directories not named `docs/`. Thanks for the hint to set `type` in the frontmatter. An even better solution for me was to set in the `_index.md` file of the top directories:
```
---
title: User's doc
cascade:
  type: docs
---
```
So that the type is automatically passed down to all pages under those directories.

(From https://gohugo.io/content-management/front-matter/#predefined)

**Reply by imfing** [reply]
@billux Thanks for reminding us the `cascade` option! This would make it much simpler to change the entire section behavior.
